Dockerfile

```diff
@@ -6,17 +6,17 @@ ADD demo/project /runner/project
 ADD demo/env /runner/env
 ADD demo/inventory /runner/inventory
 
+# Install the collection for ansible-runner integrations
+ADD ansible_runner/lib/ansible_collections/runner /usr/share/ansible/collections/ansible_collections/runner
+
 # UNDO Before 2.0 Release:
-# Install Ansible and Runner
-#ADD https://releases.ansible.com/ansible-runner/ansible-runner.el8.repo /etc/yum.repos.d/ansible-runner.repo
-#RUN dnf install -y ansible-runner
+# Install Ansible
 RUN dnf install -y python3-pip rsync openssh-clients sshpass glibc-langpack-en git \
     https://github.com/krallin/tini/releases/download/v0.19.0/tini_0.19.0-amd64.rpm && \
     rm -rf /var/cache/dnf
 
 RUN dnf install -y gcc python3-devel
-RUN pip3 install bindep https://github.com/ansible/ansible/archive/devel.tar.gz \
-    https://github.com/ansible/ansible-runner/archive/devel.tar.gz
```

Comment:
@matburt I've updated this to work in the containerized world, and it's passing some cursory integration tests I've ran locally (requires building image, so CI isn't equipped to test that).

This line shows the outright removal of the `ansible-runner` install. This is replaced by an `ADD` layer above that adds the collection directory with the callback, plus adding `psutil` directly.

The bigger picture is that we can package and vendor the collection separately. So ultimately, this could be handled in `ansible-builder` so that instead of enumerating requirements of the base image, we can just say that the manifest of collections needs to include the runner collection in order to be usable by runner.